### DISCUSSION TITLE
Table element for RTE component

### DISCUSSION THREAD

**Discussion by koddr** [mantinedev/mantine#2131] [Feature requests]
Hello,

Thanks for your product, especially the [rich text editor](https://mantine.dev/others/rte/)! But I would like to add another good function-button: **create tables**. This is needed in many projects with RTE, as a basic text editor for articles/blogs/etc.

If you have the ability (and need) to add such a button, I'd be very happy!

**Comment by rtivital** [comment]
We will probably add this feature in 6.0 release, not soon

**Reply by jiangbodev** [reply]
please consider add this..

**Reply by rtivital** [reply]
You can do that with tip tap extensions
